scripts/exregional_run_post.sh

```diff
@@ -240,14 +240,8 @@ if [ ${len_fhr} -eq 2 ]; then
 elif [ ${len_fhr} -eq 3 ]; then
   if [ "${fhr:0:1}" = "0" ]; then
     post_fhr="${fhr:1}"
-# What should happen in the "else" case?  Would just setting post_fhr to 
-# fhr work?  Need to test.
   else
-    print_err_msg_exit "\
-The \${fhr} variable contains a 3-digit integer whose first digit is not
-0.  In this case, it is not clear how to set the variable post_fhr used
-in constructing the grib2 file names generated by UPP:
-  fhr = \"$fhr\""
+    post_fhr="${fhr}"
```

Comment:
A comment here on WHY this is necessary might help the next person from going down the rabbit hole you described in personal communications.